packages-exp/auth-exp/src/core/auth/initialize.ts

```diff
@@ -24,7 +24,31 @@ import { _fail } from '../util/assert';
 import { _getInstance } from '../util/instantiator';
 import { AuthImpl } from './auth_impl';
 
-/** @public */
+/**
+ * Initializes an Auth instance with fine-grained control over
+ * {@link Dependencies}.
+ *
+ * @remarks
+ *
+ * This function allows more control over the Auth instance than
+ * {@link getAuth}. `getAuth` will use platform-specific defaults to supply
+ * the {@link Dependencies}. In general, `getAuth` is the easiest way to
+ * initialize Auth and will work in most cases. Use `initializeAuth` if you
+ * need control over which persistence layer is used, or to minimize bundle
+ * size if you're not using either `signInWithPopup` or `signInWithRedirect`.
+ *
+ * For example, if your app only uses anonymous accounts and you only want
+ * accounts saved for the current session, you would initialize Auth thus:
```

Comment:
Suggest just commanding and replacing "thus" with words describing the difference (thus is a little over formal for Google).

Like, "If your app uses only anonymous . . . , initialize it with <correct words>:"

Reply:
Done